dist/index.js

```diff
@@ -93,13 +93,26 @@ const zitiHttpRequestData = async (req, buf) => {
   });
 };
 
+function keyValuePairLinesToObj (string) {
```

Comment:
add a function to parse the action's input field `data` as a multi-line string with key=value per line